.travis.yml

```diff
@@ -15,7 +15,8 @@ script:
   - FORCE_COLOR=1 npx react-jsx-i18n extract --ext jsx indico/web/client/ indico/modules/ > /dev/null
   - isort -rc --diff --check-only indico/
   - python bin/maintenance/update_backrefs.py --ci
-  - npx eslint --ext .js --ext .jsx indico/modules/rb/ indico/modules/events/logs/ indico/web/client/js/react/
+  - npx eslint --ext .js --ext .jsx indico/modules/rb/ indico/modules/events/logs/ indico/web/client/js/react/ \
```

Comment:
no need for the backslash (and it's breaking the build since it remains there as a literal backslash)